proto/build.gradle

```diff
@@ -0,0 +1,47 @@
+description = 'OpenTelemetry Proto'
+
+apply plugin: 'com.google.protobuf'
+
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
+    }
+}
+
+dependencies {
+    api project(':opentelemetry-api')
+
+    implementation "com.google.protobuf:protobuf-java:${protobufVersion}",
```

Comment:
These 2 can be defined in the libraries to be consistent with other dependencies.